Jenkinsfile

```diff
@@ -246,6 +246,25 @@ pipeline {
                                             result = 'FAILURE'
                                             pullRequestSetResult('failure', "jenkins/provision_${backend}", 'Some test cases are failed')
                                         }
+                                        if (pullRequestContainsLabels("test-provision-reuse,test-provision-${backend}-reuse")) {
```

Comment:
Is it intentional that new `*-reuse` labels will be applied only if non-reuse one is also applied?

I think that it will be better to have it be self-sufficient.

Reply:
Updated conditions so that reuse flow can be triggered by *reuse label only.